### PR TITLE
[TypeScript][Virtual Assistant] Implement changes in Virtual Assistant to 1.0 release

### PR DESCRIPTION
**Pull request by Batta32**
### Purpose
*What is the context of this pull request? Why is it being done?*
As C# implementation is more advanced than TypeScript, we took [1.0](https://github.com/microsoft/botframework-solutions/releases/tag/v1.0) release as a point to check the changes to be migrated in Virtual Assistant.

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*
For **Virtual Assistant** functionality:
- Update `Adapters` folder
- Update `root` folder
- Update `Bots` folder
- Update `Deployment` folder
- Update `Dialogs` folder
- Update `Responses` folder
- Update `Services` folder
- Update `TokenExchange` folder
- Update `Tests` folder
- Fix `ESLint` issues

### Tests
*Is this covered by existing tests or new ones? If no, why not?*
All the tests are passing successfully validating the implemented solution.

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*
\-
#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [X] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Review by lzc850612 — APPROVED** [review]

**Review by peterinnesmsft — APPROVED** [review]

**Review by peterinnesmsft — APPROVED** [review]

**Review by lzc850612 — APPROVED** [review]

File: templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/src/tokenExchange/tokenExchangeSkillHandler.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
     ResourceResponse,
     TurnContext,
     SkillConversationReference,
-    InvokeResponse
+    InvokeResponse, SkillConversationIdFactoryBase
```
Comment (peterinnesmsft) [inline templates/typescript/generator-bot-virtualassistant/generators/app/templates/sample-assistant/src/tokenExchange/tokenExchangeSkillHandler.ts]: Move to separate line

File: templates/typescript/samples/sample-assistant/src/tokenExchange/tokenExchangeSkillHandler.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import {
     ResourceResponse,
     TurnContext,
     SkillConversationReference,
-    InvokeResponse
+    InvokeResponse, SkillConversationIdFactoryBase
```
Comment (peterinnesmsft) [inline templates/typescript/samples/sample-assistant/src/tokenExchange/tokenExchangeSkillHandler.ts]: Move to new line

**Review by peterinnesmsft — DISMISSED** [review]
Couple of minor changes.

**Comment by Batta32** [comment]
Thanks for the feedback @peterinnesmsft , we already updated the PR with the changes!

**Review by lauren-mills — APPROVED** [review]

**Pull request merged**